lingua_franca/lang/parse_ca.py

```diff
@@ -337,9 +337,10 @@ def date_found():
                 minAbs or secOffset != 0
             )
 
-    if text == "" or not anchorDate:
+    if text == "":
```

Comment:
"Bikeshedding (also referred to as the law of triviality) describes a phenomenon where people spend a relatively large amount of time, energy, and other resources dealing with relatively minor issues.

The concept of bikeshedding has important implications, both in terms of personal productivity as well as in terms of group efficiency, since it can help identify situations where people waste valuable resources on trivial matters, while also neglecting important issues which require their attention."